pages/common/routers/confirm.js

```diff
@@ -21,16 +15,17 @@ module.exports = ({
 
   const _configure = (req, res, next) => {
     req.form = req.form || {};
-    req.form.id = req[model];
+    req.model = req.model || {};
+    req.model.id = req.model.id || `new-${model}`;
     req.form.schema = schema;
     return configure(req, res, next);
   };
 
   const _processQuery = (req, res, next) => {
     const { clear, edit } = req.query;
     if (clear) {
-      delete req.session.form[req.form.id];
-      const re = new RegExp(`/${req.form.id}/edit/confirm`);
+      delete req.session.form[req.model.id];
+      const re = new RegExp(`/${req.model.id}/edit/confirm`);
```

Comment:
This needs fixing - the id, and the `edit` bit are both incorrect.

Reply:
fixed - redirects to '/'